mix.exs

```diff
@@ -21,7 +21,7 @@ defmodule ExDoc.Mixfile do
   end
 
   defp deps do
-    [{:earmark, "~> 1.1"},
+    [{:earmark, "~> 1.2"},
```

Comment:
Let's not bump the requirement unless we have to because people may still rely on earlier earmark versions if using earlier elixir versions. What do you think?

Reply:
Couldn't they use the previous version of ex_doc in this case?